packages/amplify_auth_cognito/ios/Classes/FlutterSignInResult.swift

```diff
@@ -59,7 +65,7 @@ func setCodeDeliveryDetails(res: AmplifyOperation<AuthSignInRequest, AuthSignInR
           if case let .confirmSignInWithSMSMFACode(deliveryDetails, _) = signInResult.nextStep {
             if case let .sms(e) = deliveryDetails.destination {
               deliveryMap["destination"] = e! as String
-              deliveryMap["attributeName"] = ""
+              deliveryMap["attributeName"] = deliveryDetails.attributeKey != nil ? "" : nil
```

Comment:
QQ: Kind of confused why we want to explicitly set an empty string here and why we need to discern between that and nil

Reply:
Hmm... I think something did not get committed properly.  This is old.

Reply:
Fixed this - we do not need the value at all.